components/i2c/sht20/definition.json

```diff
@@ -0,0 +1,5 @@
+{
+  "displayName": "sht20",
+  "i2cAddresses": [ "0x40" ],
+  "subcomponents": [ "ambient-temp", "ambient-temp-celcius", "humidity" ]
```

Comment:
@tyeth  @rob-vaughn So, `ambient-temp` is in celsius. Remove `ambient-temp-celcius. Use `ambient-temp-fahrenheit` instead